File: algo-c-to-_/examples/zeta.lua
--
--	from src/zeta.c
--
--	double zeta(double)	to	riemannZeta
--

local M = require 'zeta'

local riemannZeta = M.riemannZeta

do
	local zeta = riemannZeta()

	for x=-4,0 do
		print(("zeta(%2d) : %.15f"):format(x, zeta(x)))
	end

	print()

	for x=2,20 do
		print(("zeta(%2d) : %.15f"):format(x, zeta(x)))
	end
end

File: algo-c-to-_/examples/zeta_.py
#
#	from src/zeta.c
#
#	double zeta(double)	to	riemannZeta
#

from zeta import riemannZeta

zeta = riemannZeta()

for x in range(-4, 1):
    print("zeta({0:2}) : {1:.15f}".format(x, zeta(x)))

print()

for x in range(2, 21):
    print("zeta({0:2}) : {1:.15f}".format(x, zeta(x)))
